Scripts/SearchIncidentsV2/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
+- Fixed an issue where special characters returned an error.
```

Comment:
so this is now a new script, do we need changelog for that?
we will add in the rn about the breaking changes regardless

Reply:
since it was released, you will need change log...

Reply:
leave the changelog